src/components/common/GridContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC, ReactNode } from 'react'
+import styled from '@emotion/styled'
+
+import { theme } from '../../theme'
+
+const Container = styled.div<{ columns: number }>`
+  display: grid;
+  grid-template-columns: ${({ columns }) => `repeat(${columns}, 1fr)`};
```

Comment:
Can you quickly explain what these `grid` related properties are and why they are needed?